content/ja/docs/tasks/job/automated-tasks-with-cron-jobs.md

```diff
@@ -0,0 +1,175 @@
+
+## {{% heading "prerequisites" %}}
+
+* {{< include "task-tutorial-prereqs.md" >}}
+
+<!-- steps -->
+
+## CronJobを作成する
+
+CronJobには設定ファイルが必要です。次の例のCronJobの`.spec`は、現在の時刻とhelloというメッセージを1分ごとに表示します。
+
+{{< codenew file="application/job/cronjob.yaml" >}}
+
+次のコマンドで例のCronJobを実行します。
+
+```shell
+kubectl create -f https://k8s.io/examples/application/job/cronjob.yaml
+```
+
+出力は次のようになります。
+
+```
+cronjob.batch/hello created
+```
+
+CronJobを作成したら、次のコマンドで状態を取得します。
+
+```shell
+kubectl get cronjob hello
+```
+
+出力は次のようになります。
+
+```
+NAME    SCHEDULE      SUSPEND   ACTIVE   LAST SCHEDULE   AGE
+hello   */1 * * * *   False     0        <none>          10s
+```
+
+コマンドの結果からわかるように、CronJobはまだスケジュールされておらず、まだ何のJobも実行していません。約1分以内にJobが作成されるのを見てみましょう。
+
+```shell
+kubectl get jobs --watch
+```
+
+出力は次のようになります。
+
+```
+NAME               COMPLETIONS   DURATION   AGE
+hello-4111706356   0/1                      0s
+hello-4111706356   0/1           0s         0s
+hello-4111706356   1/1           5s         5s
+```
+
+"hello"CronJobによってスケジュールされたJobが1つ実行中になっていることがわかります。Jobを見るのをやめて、再度CronJobを表示して、Jobがスケジュールされたことを確認してみます。
+
+```shell
+kubectl get cronjob hello
+```
+
+出力は次のようになります。
+
+```
+NAME    SCHEDULE      SUSPEND   ACTIVE   LAST SCHEDULE   AGE
+hello   */1 * * * *   False     0        50s             75s
+```
+
+CronJob`hello`が、`LAST SCHEDULE`で指定された時間にJobを正しくスケジュールしたことが確認できるはずです。現在、activeなJobの数は0です。つまり、Jobは完了または失敗したことがわかります。
+
+それでは、最後にスケジュールされたJobの作成と、Podの1つの標準出力を表示してみましょう。
+
+{{< note >}}
+Jobの名前とPodの名前は異なります。
+{{< /note >}}
+
+```shell
+# "hello-4111706356" の部分は、あなたのシステム上のJobの名前に置き換えてください。
+pods=$(kubectl get pods --selector=job-name=hello-4111706356 --output=jsonpath={.items[*].metadata.name})
+```
+
+Podのログを表示します。
+
+```shell
+kubectl logs $pods
+```
+
+出力は次のようになります。
+
+```
+Fri Feb 22 11:02:09 UTC 2019
+Hello from the Kubernetes cluster
+```
+
+## CronJobの削除
+
+CronJobが必要なくなったときは、`kubectl delete cronjob <cronjob name>`で削除します。
+
+```shell
+kubectl delete cronjob hello
+```
+
+CronJobを削除すると、すべてのJobと、そのJobが作成したPodが削除され、追加のJobの作成が停止されます。Jobの削除について詳しく知りたい場合は、[ガベージコレクション](/ja/docs/concepts/workloads/controllers/garbage-collection/)を読んでください。
+
+## CronJobのSpecを書く
+
+すべてのKubernetesの設定と同じように、CronJobにも`apiVersion`、`kind`、`metadata`のフィールドが必要です。設定ファイルの扱い方についての一般的な情報については、[アプリケーションのデプロイ](/ja/docs/tasks/run-application/run-stateless-application-deployment/)と[kubectlを使用してリソースを管理する](/ja/docs/concepts/overview/working-with-objects/object-management/)を読んでください。
+
+CronJobの設定には、[`.spec`セクション](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status)も必要です。
+
+{{< note >}}
+CronJobの特に`spec`へのすべての修正は、それ以降の実行にのみ適用されます。
+{{< /note >}}
+
+### Schedule
+
+`.spec.schedule`は、`.spec`には必須のフィールドです。`0 * * * *`や`@hourly`などの[Cron](https://en.wikipedia.org/wiki/Cron)形式の文字列を取り、Jobの作成と実行のスケジュール時間を指定します。
+
+フォーマットには`vixie cron`のステップ値(step value)も指定できます。[FreeBSDのマニュアル](https://www.freebsd.org/cgi/man.cgi?crontab%285%29)では次のように説明されています。
```

Comment:
I'd write:
```suggestion
フォーマットにはVixie cronのステップ値(step value)も指定できます。[FreeBSDのマニュアル](https://www.freebsd.org/cgi/man.cgi?crontab%285%29)では次のように説明されています。
```
Vixie cron is named after Paul Vixie, who wrote it at Digital Equipment Corporation.

Reply:
Oh no, I didn't know him until now. I've just learned he has played a huge role on the Internet.